### DISCUSSION TITLE
Add Existing Sensor to New Manager

### DISCUSSION THREAD

**Discussion by Mfisher925** [Security-Onion-Solutions/securityonion#7848] [Unsupported Versions] [answered]
Hello,

We had a Security Onion distributed deployment with a single Manager node, 2 search nodes and remote sensor nodes.

We recently had a hardware failure on our equipment hosting our Manager and Search Nodes.  We still have a number of remote sensor nodes throughout our network that I would like to re-enroll/add to a new Manager and Search Node deployment.

Is this possible?  I can't find a reference to that specific scenario in the documentation.

Thanks,

**Comment by dougburks** [comment, marked as answer]
You should be able to re-run Setup:
https://docs.securityonion.net/en/2.3/installation.html

**Reply by Mfisher925** [reply]
Thanks! I did try that but I kept getting "Install Had a Problem" errors after running so-setup with identical settings/hardware.  There weren't any errors in logs either.  I ended up re-deploying the forward nodes from the ground up.
